helix-term/src/commands.rs

```diff
@@ -3949,6 +3950,25 @@ fn transpose_view(cx: &mut Context) {
     cx.editor.transpose_view()
 }
 
+fn goto_window_idx(cx: &mut Context) {
+    cx.editor.show_window_ids = true;
+
+    cx.on_next_key(move |cx, event| {
+        if let KeyEvent {
+            code: KeyCode::Char(ch),
+            ..
+        } = event
+        {
+            let idx_int = ch.to_digit(10).unwrap();
```

Comment:
This will panic if you hit something outside of 0-9. Instead we should say `cx.editor.set_error("Window index must be 1-9")` if the character isn't in 1-9.

Reply:
There's some slightly similar prior work around parsing usizes you can use as a reference https://github.com/helix-editor/helix/pull/561/files#diff-d782c07f22492bcae17333aba5dc5b840fb30317727e71267ebf4863f84787fdR1458-R1462